charts/keycloakx/ci/h2-values.yaml

```diff
@@ -32,5 +38,31 @@ secrets:
       user: admin
       password: secret
 
+http:
+  relativePath: "/"
```

Comment:
I've confirmed that even default path `/auth` also pass the test.

https://github.com/codecentric/helm-charts/blob/cab3997511239923a030333e006db569239d2ca3/charts/keycloakx/values.yaml#L394-L396

So I think it's better not to set it individually.